src/capi/TuriCreate.h

```diff
@@ -874,26 +874,27 @@ tc_variant* tc_function_call(
 
 // Single SArray view (`.show` on an SArray)
 tc_plot* tc_plot_create_1d(const tc_sarray* sa,
-                           const char *title,
-                           const char *x_axis_title,
-                           const char *y_axis_title,
+                           const char* title,
+                           const char* x_axis_title,
+                           const char* y_axis_title,
+                           tc_parameters* params,
```

Comment:
Let's make this `const tc_parameters*`, then good to go.

Reply:
Thanks, will do.